server/routes/users/auth.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const passport = require('passport');
+
+const router = express.Router();
+
+router.post('/login', passport.authenticate('local', {
```

Comment:
The last callback method won't fire here if you have the options params that indicates what should happen after authentication.